examples/calico-cert.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: "giantswarm.io/v1"
+kind: Certificate
+metadata:
+  name: "uo91f-calico"
+  namespace: "default"
+spec:
+  clusterID: "uo91f"
+  clusterComponent: "calico"
+  commonName: "calico.uo91f.g8s.eu-west-1.aws.test.private.giantswarm.io"
```

Comment:
See 

- https://github.com/giantswarm/giantswarm/issues/1305
- https://github.com/giantswarm/giantswarm/issues/1304

Feel free to do that in case your are blocked by it somehow. The domains for the guest cluster components have all the same structure and only differ in their sub domain. I don't know how the base domain of AWS clusters look like. And note that the kube DNS domain is a completely different thing structure wise.